graylog2-server/src/main/java/org/graylog2/bindings/providers/EsNodeProvider.java

```diff
@@ -48,43 +47,45 @@ public EsNodeProvider(ElasticsearchConfiguration configuration) {
     }
 
     @Override
-    @Singleton
```

Comment:
Is moving the `@Singleton` in front of the class actually the same as having it on the `get()` method?